content/doc/tutorials/build-a-python-app-with-pyinstaller.adoc

```diff
@@ -17,43 +17,36 @@ application using Jenkins, then this tutorial is for you.
 
 The simple Python application (which you'll obtain from a sample repository on
 GitHub) is a command line tool "add2vals" that outputs the addition of two
-values. If at least one of the values is a string, "add2vals" treats both values
-as a string and instead concatenates the values. The "add2" function in the
-"calc" library (which "add2vals" imports) is accompanied by a set of unit tests.
+values.
+If at least one of the values is a string, "add2vals" treats both values
+as a string and instead concatenates the values.
```

Comment:
```suggestion
If at least one of the values is a string, "add2vals" instead treats both values
as a string and concatenates the values.
```
Suggestion to move "instead" so that it makes more grammatical sense